### ISSUE TITLE
Browse button in the window "Select files" opens no window

### ISSUE THREAD

**Issue by jonasstein** [JabRef/jabref#2969]
What I did: I tried to link a PDF to one entry. In the menu select files I clicked on Browse.
No window opened. Not even in the background. I could not click other buttons in Jabref anymore and had to kill it.

System: Linux Mint, Mate 18.1, Filesystem: ext4 + samba mounts for some folders

![selection_082](https://user-images.githubusercontent.com/298682/27859891-90286fcc-617a-11e7-8971-dc9937ff97fe.png)

```
JabRef 4.0.0-dev--snapshot--2017-07-03--master--6897b10fb
Linux 4.4.0-78-generic amd64 
Java 1.8.0_131

```

**Comment by Siedlerchr** [comment]
Related to https://github.com/JabRef/jabref/issues/2930#issuecomment-311050976

**Comment by lenhard** [comment]
@Siedlerchr You fixed the issue you linked above. Did this also fix this one?

At least I tried to link a PDf to an entry right now and it works.

**Comment by Siedlerchr** [comment]
Yes, this should be fixed by #3079
